docs/interfaces/postgres.rst

```diff
@@ -363,6 +363,13 @@ JDBC
 Limitations
 '''''''''''
 
+- Versions ``42.7.5`` and ``42.7.6`` do not support metadata methods in general.
+  If possible, it is recommended to avoid these versions.
+
+- Version ``42.7.7`` and above only works with CrateDB ``6.0`` and above. If you are
```

Comment:
I thought these versions in general also work with <6.0, with the exception of some metadata methods?

Reply:
I have changed the wording a bit in the hope that it is clear.